src/Uno.UI/Microsoft/UI/Xaml/Controls/TabView/TabView.cs

```diff
@@ -551,7 +551,9 @@ private void OnListViewLoaded(object sender, RoutedEventArgs args)
 					TabItems = lvItems;
 				}
 
-				if (ReadLocalValue(SelectedItemProperty) != DependencyProperty.UnsetValue)
+				// ReadLocalValue() is not returning UnsetValue even though SelectedItem is not yet set
+				if (ReadLocalValue(SelectedItemProperty) != DependencyProperty.UnsetValue
```

Comment:
Because of this issue, https://github.com/unoplatform/uno/pull/18492 the code is taking a different path, causing one of the Runtime Tests to fail.